air-quality-ui/src/components/single-city/AverageComparisonChart.tsx

```diff
@@ -0,0 +1,53 @@
+import ReactECharts from 'echarts-for-react'
+
+import classes from './AverageComparisonChart.module.css'
+import { convertToLocalTime, xAxisFormat } from '../../services/echarts-service'
+import { ForecastResponseDto } from '../../services/types'
+
+function getChartOptions(forecastData?: ForecastResponseDto[]) {
```

Comment:
It's actually difficult. Echarts has an issue where it doesn't really work with Jest and node (you'll need to mock a lot of things it requires and end up not actually testing anything). And then even if it did, it's probably better for Playwright to test it rather than Jest.

I've added a test to make sure it renders - but it's really only testing that a mocked chart exists at the moment as the component is only a chart